07.bitstream.semantics.md

```diff
@@ -2166,6 +2166,8 @@ for each tile down the image.
 **height_in_sbs_minus_1** specifies the height of a tile minus 1 in units of superblocks.
 
 **maxTileHeightSb** specifies the maximum height (in units of superblocks) that can be used for a tile (to avoid making tiles with too much area).
+If uniform_tile_spacing_flag is equal to 0, the maximum height for a tile is calculated such that the widest tile has an maximum area of one half 
```

Comment:
To make it clear that this is not a normative change I would recommend putting this in a note.  You can put it in a note by adding: 
**Note:** 
Before the text and:
{:.alert .alert-info }
after the text.  (See line 2155 for an example.)

Also, I would change "an maximum" to "a maximum".